assets/scss/00-base/_colors.scss

```diff
@@ -25,6 +25,7 @@ $c-granite-gray: #535353;
 $c-overcast-gray: #DCDCDC;
 $c-platinum-gray: #F2F2F2;
 $c-atlantic-gray: #707070;
+$c-very-light-gray: #CCCCCC;

```

Comment:
we should look at this - it should be added to the gray variable color scale in this file as well.

Reply:
good catch, so the structure of base color variables are not meant to be used direction, we want to pass these to a greyscale variable. Let's do follow up later